himl/config_generator.py

```diff
@@ -139,6 +139,8 @@ def dict_constructor(loader, node):
 
         Dumper.add_representer(str, SafeRepresenter.represent_str)
 
+        yaml.representer.BaseRepresenter.represent_scalar = ConfigGenerator.custom_represent_scalar
```

Comment:
@ecojan. Eugen, I'm wondering if this is going to break something for users that importing the `himl` package into their project, given that its applying the representer globally, on `yaml`. To that end, do you see any way of making it possible to disable this representer, via a flag or something?

Reply:
@costimuraru I see your point and I completely, this should be enabled via a flag, let me push an update and I will ping you for a review!